src/aliases.rs

```diff
@@ -56,6 +56,7 @@ alias! {
     U384 B384 (384, 6);
     U448 B448 (448, 7);
     U512 B512 (512, 8);
+    U784 B784 (784, 12);
```

Comment:
the `LIMBS` param needs to be 13. `64 * 12 == 768`, which will not quite fit 784 bits

Reply:
yeah it should be U768 and B768, sorry

Reply:
ah cool. mind pushing a commit to change that?